.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,11 @@
+---
+name: Question
+about: Please raise your question on our community platforms. https://www.thethingsnetwork.org/forum/ or https://thethingsnetwork.slack.com
+
+---
+
+:rotating_light: :stop_sign: This issue tracker is not for questions :stop_sign: :rotating_light:
+
+Please raise your question on our community platforms:
+- The forum: https://www.thethingsnetwork.org/forum/
```

Comment:
I personally think "Forum" is better than "The Forum" but this is only a nitpick.

Reply:
good point though.